docs/crates/db.md

```diff
@@ -180,19 +168,28 @@ The `TXMut` type can be any type that implements the `DbTxMut` trait, which prov
 
 ```rust ignore
 /// Read write transaction that allows writing to database
-pub trait DbTxMut: for<'a> DbTxMutGAT<'a> {
+pub trait DbTxMut: Send + Sync {
+    /// Read-Write Cursor type
+    type CursorMut<T: Table>: DbCursorRW<T> + DbCursorRO<T> + Send + Sync;
+    /// Read-Write DupCursor type
+    type DupCursorMut<T: DupSort>: DbDupCursorRW<T>
+    + DbCursorRW<T>
+    + DbDupCursorRO<T>
+    + DbCursorRO<T>
+    + Send
+    + Sync;
```

Comment:
needs indentation

Reply:
unclear, it is formatted

Reply:
```suggestion
        + DbCursorRW<T>
        + DbDupCursorRO<T>
        + DbCursorRO<T>
        + Send
        + Sync;
```

Reply:
Run rustfmt on this https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=e1cd2ad7a9444280d2fe6ceb08158412

Reply:
ah my bad, that's a comment on the md file...